src/Microsoft.AspNetCore.Razor.Language/IRazorImportExpander.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public interface IRazorImportExpander : IRazorProjectEngineFeature
```

Comment:
Curious, how will we add default imports in the new world?

Reply:
Talked offline. It's going to be part of the `IRazorImportFeature`.